v3.0/reference/tispark.md

```diff
@@ -263,3 +263,7 @@ A: You can ignore this warning. It occurs because Spark tries to load two nonexi
 Q: What can I do if `java.sql.BatchUpdateException: Data Truncated` is returned when executing SQL statements using TiSpark?
 
 A: This error occurs because the length of the data written exceeds the length of the data type defined by the database. You can check the field length and adjust it accordingly.
+
+Q: What can I do if `Error：java.io.InvalidClassException: com.pingcap.tikv.region.TiRegion; local class incompatible: stream classdesc serialVersionUID ...` is returned when TiSpark is executing a Spark task?
+
+A: The error message shows a `serialVersionUID` conflict, which occurs because you have used `class` and `TiRegion` of different versions. Because `TiRegion` only exists in TiSpark, it might be that multiple TiSpark packages are used. To fix this error, you need to make sure that the versions of TiSpark dependency are consistent for each node in the cluster.
```

Comment:
```suggestion
A: The error message shows a `serialVersionUID` conflict, which occurs because you have used `class` and `TiRegion` of different versions. Because `TiRegion` only exists in TiSpark, multiple versions of TiSpark packages might be used. To fix this error, make sure the version of TiSpark dependency is consistent among all nodes in the cluster.
```